images/dcos/Makefile

```diff
@@ -29,7 +29,7 @@ endif
 # compute the Docker image tag
 # NOTE: As VERSION, do not persist DOCKER_IMAGE
 DOCKER_REPO = $(DOCKER_ORG)/kubernetes
-DOCKER_IMAGE = $(DOCKER_REPO):$(VERSION)$(POSTFIX)
+DOCKER_IMAGE = $(DOCKER_REPO)$(VERSION)$(POSTFIX)
```

Comment:
Sorry, still don't understand why this failed. According to docker docs the `:tag` notation should still work.
https://docs.docker.com/engine/reference/commandline/tag/

Is there another approach we can take to avoid losing the `:tag` information?